objetivos/leanuxcanvas.md

```diff
@@ -1,41 +1,72 @@
 # Lean UX canvas
 
 <!--
-Utiliza la plantilla de este documento para recoger vuestro Lean UX Canvas en el proyecto. 
-Escribid solamente debajo de cada título. **No modifiquéis el código HTML, ya que si lo 
-hacéis la tabla no se mostrará correctamente**.
--->
+Utiliza la plantilla de este documento para recoger vuestro Lean UX Canvas en el proyecto. Escribid solamente debajo de cada título. **No modifiquéis el código HTML, ya que si lo hacéis la tabla no se mostrará correctamente**.
+-->  
 
 <table markdown="1"><tbody><tr><td markdown="1">
 
 **Problema**
 
+No estar viendo la TV o escuchando la radio el día en el que se emite el programa o el podcast o no comprar el periódico el día en el que se publica la revista. La página web también va a ofrecer la opción de consumir más contenido del que ha aparecido en el programa o en la revista. 
+
 </td><td rowspan=2 markdown="1">
 
-**Ideas de soluciones**
+**Soluciones**
+
+- Posibilidad de consumir los tres productos en cualquier momento
+- Facilitar el contenido entero del material que aparece en los medios (allí solo aparecerá una parte de todo el proceso) -> “para los más curiosos”, ”por si te lo perdiste”, ”más sobre…”
+- Posibilitar al consumidor la opción de compartir el contenido
 
 </td><td markdown="1">
 
 **Resultados**
 
+- Número de visitas
+- Interacciones
+- Cantidad de “me gusta”
+- Cantidad de veces que se ha compartido 
+- Número de visualizaciones o escuchas del contenido audio y audiovisual
+
+
 </td></tr><tr><td markdown="1">
 
 **Usuarios**
 
+Nuestro público objetivo serán jóvenes de 18 a 30 años.
+
 </td><td markdown="1">
 
-**Beneficios para los usuarios**
+**Resultados y beneficios para los usuarios**
+
+- Por falta de información sobre el tema que trataremos
+- El beneficio que tratamos de ofrecer es la mezcla de información sobre política y entretenimiento
+- Sabremos si hemos conseguido nuestro objetivo (o no) observando la interactividad de nuestro contenido en las redes sociales. Si se  han obtenido muchas interacciones, sabremos que ha sido consumido por una gran cantidad de personas
+
 
 </td></tr><tr><td markdown="1">
 
 **Hipótesis**  
 
+2. Creemos que obtendremos buenos resultados en la medición cuantitativa del número de visitas, interacciones, ”me gusta”… si hacemos una página web con usabilidad.
+3. Creemos que los jóvenes consumirán nuestro producto si hacemos una página web que se adecue a sus necesidades
+4. Creemos que los usuarios se informarán y se entretendrán si conseguimos crear contenido informativo de manera atractiva
+5. Creemos que los usuarios podrán consumir todo el contenido en cualquier lugar y a cualquier hora, además de compartirlo, si creamos un soporte web para alojar todos los contenidos y sea accesible en la web
+
```

Comment:
~~No entiendo a qué corresponde la numeración de estas hipótesis, ni por qué empieza en 2.~~

Habéis formulado una hipótesis por cada elemento (casillas 2, 3, 4 y 5), y en realidad se trata de **combinar** las suposiciones de las cuatro casillas para formar una (o varias) hipótesis. En vuestro caso, por ejemplo:

- Creemos que obtendremos **xx visitas/un xx% de la audiencia en directo en visitas** si **los jóvenes de entre 18 y 30 años** consiguen **acceder al contenido fuera del horario de emisión/soporte físico** a través de **las referencias durante la emisión en directo/soporte físico en papel**.
- Creemos que podremos **aumentar las visitas a través de la web hasta un xx% más que la audiencia en directo/en soporte físico** si **los jóvenes de entre 18 y 30 años** consiguen **acceder al contenido** con **la función de compartir el contenido que haya utilizado algún contacto suyo**.
- Creemos que conseguiremos **saciar la curiosidad de quienes han consumido en directo/en papel** si **los jóvenes de entre 18 y 30 años** acceden **al contenido adicional ofrecido** mediante **los materiales en bruto o tipo "saber más"**.
- ...